demos/real_finger_print_pos.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import time
+import numpy as np
+
+import py_finger
+import py_real_finger
+
+
+finger_data = py_finger.Data()
+finger_backend = py_real_finger.create_real_finger_backend("can0", "can1",
+                                                           finger_data)
+finger = py_finger.Frontend(finger_data)
+
+
+finger_backend.calibrate()
+
+# FIXME this makes the application hang
```

Comment:
I think it would be nice to be able to already get observations before sending actions (simple use case: I want to know where I am before I start moving).  But this is probably out of scope for this PR.

Reply:
ok, we can disucss this in person, that's easier. but definitely out of scope here.